internal/process/vms.go

```diff
@@ -239,3 +245,37 @@ func createVirtualMachineSamples(config *config.Config) {
 		}
 	}
 }
+
+func computeFullHostname(vm *mo.VirtualMachine) string {
```

Comment:
perhaps adding some comments regarding the limitations for this to work

Reply:
also that the first valid of the array is returned

Reply:
Done!